lewis/adapters/epics.py

```diff
@@ -321,7 +321,7 @@ def _create_getter(self, func, *targets):
             raise RuntimeError(
                 'The function \'{}\' does not look like a getter function. A valid getter '
                 'function has no arguments that do not have a default. The self-argument of '
-                'methods does not count towards that number.')
+                'methods does not count towards that number.'.format(final_callable.__name__))
```

Comment:
True, good thing you remembered that. For the time being I don't see an urgent need to support partials and/or callable classes...I don't know about "ever". Does it work for stream?